crates/db/migrations/20230113195325_quests.sql

```diff
@@ -22,6 +22,13 @@ CREATE TABLE IF NOT EXISTS events (
   timestamp TIMESTAMP NOT NULL DEFAULT now()
 );
 
+CREATE TABLE IF NOT EXISTS abandoned_quests (
```

Comment:
The same question as here: https://github.com/decentraland/quests/pull/71#discussion_r1167180132

Besides that, why not add a field into the `quest_instaces` table called `is_abandoned` or `is_active` instead of creating a new table? It's more straightforward and it allows simpler queries.

Reply:
When doing soft deletes we have both options, adding a column or a separated table, but I like the idea of only having `INSERT`s and avoiding `UPDATE`s that can quickly convert into bugs. It's a tradeoff, so then we must do extra logic when retrieving active quests/instances but I think it is worth it.